### ISSUE TITLE
Game is crashing for a lot of people always in the same part of the game

### ISSUE THREAD

**Issue by ingeninovel** [renpy/renpy#5247]
I've launched a game and for the most part of the users there is no issue. But for some the game is crashing always in the same part of the game, at least 6 people have already reported. Here are some logs

Sat Jan 6 21:46:37 2024
Windows-10-10.0.19045
Ren'Py 8.1.3.23091805

Bootstrap to the start of init.init. took 0.05s
Early init. took 0.00s
Loader init. took 0.03s
Loading error handling. took 0.07s
Loading script. took 0.39s
Loading save slot metadata. took 0.04s
Loading persistent. took 0.00s
Set script version to: (8, 1, 3)
Running init code. took 0.08s
Loading analysis data. took 0.04s
Analyze and compile ATL. took 0.04s
Reloading save slot metadata. took 0.03s
Index archives took 0.00s
Dump and make backups. took 0.00s
Cleaning cache. took 0.00s
Making clean stores. took 0.00s
Initial gc. took 0.05s
DPI scale factor: 1.000000
nvdrs: Loaded, about to disable thread optimizations.
nvdrs: b"Couldn't load nvlib." (can be ignored)
Creating interface object. took 0.00s
Cleaning stores took 0.00s
Init translation took 0.08s
Build styles took 0.00s
Load screen analysis took 0.05s
Analyze screens took 0.00s
Save screen analysis took 0.00s
Prepare screens took 0.10s
Save pyanalysis. took 0.00s
Save bytecode. took 0.00s
Running _start took 0.00s
Performance test:
Interface start. took 0.13s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1366, 768)
swap interval: 1 frames
Windowed mode.
Vendor: "b'ATI Technologies Inc.'"
Renderer: b'AMD Radeon(TM) Vega 3 Graphics'
Version: b'4.6.13559 Compatibility Profile Context 26.20.12036.1'
Display Info: None
Screen sizes: virtual=(1920, 1080) physical=(1366, 705) drawable=(1366, 705)
Could not open 'cache/shaders.txt':
Maximum texture size: 4096x4096

and another one

Sun Jan  7 16:39:47 2024
Windows-10-10.0.22635
Ren'Py 8.1.3.23091805
 

Bootstrap to the start of init.init. took 0.06s
Early init. took 0.01s
Loader init. took 0.05s
Loading error handling. took 0.11s
Loading script. took 1.37s
Loading save slot metadata. took 0.09s
Loading persistent. took 0.00s
Set script version to: (8, 1, 3)
Running init code. took 0.22s
Loading analysis data. took 0.05s
Analyze and compile ATL. took 0.05s
Reloading save slot metadata. took 0.06s
Index archives took 0.00s
Dump and make backups. took 0.00s
Cleaning cache. took 0.00s
Making clean stores. took 0.00s
Initial gc. took 0.06s
DPI scale factor: 1.250000
nvdrs: Loaded, about to disable thread optimizations.
nvdrs: Disabled thread optimizations.
Creating interface object. took 0.45s
Cleaning stores took 0.00s
Init translation took 0.08s
Build styles took 0.01s
Load screen analysis took 0.06s
Analyze screens took 0.00s
Save screen analysis took 0.00s
Prepare screens took 0.13s
Save pyanalysis. took 0.00s
Save bytecode. took 0.00s
Running _start took 0.00s
Performance test:
Interface start. took 0.20s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1920, 1080)
swap interval: 1 frames
Windowed mode.
Vendor: "b'ATI Technologies Inc.'"
Renderer: b'AMD Radeon(TM) Graphics'
Version: b'4.6.13587 Compatibility Profile Context 19.40.54.01 26.20.14054.1001'
Display Info: None
Screen sizes: virtual=(1920, 1080) physical=(1732, 975) drawable=(1732, 975)
Could not open 'cache/shaders.txt':
Maximum texture size: 4096x4096
Screen sizes: virtual=(1920, 1080) physical=(1920, 1080) drawable=(1920, 1080)
Could not open 'cache/shaders.txt':
Maximum texture size: 4096x4096

I can't find the issue :(

**Comment by Gouvernathor** [comment]
What's the traceback for the crash ?

**Comment by ingeninovel** [comment]
I'll ask asap. Thanks for the help!

**Comment by renpytom** [comment]
You may also want to get a stack traceback - and make sure you're on the latest versions, and that config.gl2 = True.

**Comment by ingeninovel** [comment]
[traceback.zip](https://github.com/renpy/renpy/files/13854673/traceback.zip)
the only thing they gave me was this. Sorry, I myself am fairly recent to Ren'Py. Does it help?

Where do I see that config.gl2 = True @renpytom ?

**Comment by ingeninovel** [comment]
But I'm using renpy 8.1.3 so it shouldn't be that I think.

**Comment by renpytom** [comment]
Oh, this is windows.... that's rough. Especially for 8.1.3, which has been out for months without problems like this, it seems likely that this is something game specific, or else bad drivers. 

The first thing for them to do is to update their video drivers, which appear to be over 3.5 years old.

**Comment by ingeninovel** [comment]
I'll try. One of the players said he had a new laptop a omen 15 or something like that. I googled and it was a recent machine. So I never thought of the drivers. They all had AMD gpus though. If I could get a machine to replicate the issue it would be easier

**Comment by renpytom** [comment]
Is there anything weird about the images in that part of the game? Like an image that's huge, or 0 pixels wide, or a custom shader?

**Comment by ingeninovel** [comment]
No, all 1920x1080. No custom shaders. Just has a lot of image buttons in a screen

**Comment by ingeninovel** [comment]
3 more people reported the same issue, but I only have the logs. They say it freezes for a moment and then just exits to window

Mon Jan  8 16:35:10 2024
Windows-10-10.0.22621
Ren'Py 7.6.3.23091805
 

Bootstrap to the start of init.init. took 0.07s
Early init. took 0.00s
Loader init. took 0.01s
Loading error handling. took 0.05s
Loading script. took 0.79s
Loading save slot metadata. took 0.07s
Loading persistent. took 0.00s
Set script version to: (7, 6, 3)
Running init code. took 0.14s
Loading analysis data. took 0.04s
Analyze and compile ATL. took 0.03s
Reloading save slot metadata. took 0.09s
Index archives took 0.00s
Dump and make backups. took 0.00s
Cleaning cache. took 0.00s
Making clean stores. took 0.00s
Initial gc. took 0.05s
DPI scale factor: 1.250000
nvdrs: Loaded, about to disable thread optimizations.
nvdrs: Disabled thread optimizations.
Creating interface object. took 0.26s
Cleaning stores took 0.00s
Init translation took 0.06s
Build styles took 0.00s
Load screen analysis took 0.03s
Analyze screens took 0.00s
Save screen analysis took 0.00s
Prepare screens took 0.09s
Save pyanalysis. took 0.00s
Save bytecode. took 0.00s
Running _start took 0.00s
Performance test:
Interface start. took 0.13s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1920, 1080)
swap interval: 1 frames
Fullscreen mode.
Vendor: 'ATI Technologies Inc.'
Renderer: 'AMD Radeon(TM) Graphics'
Version: '4.6.13587 Compatibility Profile Context 19.40.54.01 26.20.14054.1001'
Display Info: None
Screen sizes: virtual=(1920, 1080) physical=(1920, 1080) drawable=(1920, 1080)
Could not open 'cache/shaders.txt':
Maximum texture size: 4096x4096

**Comment by ingeninovel** [comment]
I tried compiling in a older renpy version to see if anything helped and gave that person to test, that is their log

**Comment by danac78** [comment]
8.1.3 and 7.6.3 are identical (for the most part) except for the fact that 7.6.3 is Python 2 and 8.1.3 is Python 3.

**Comment by renpytom** [comment]
I don't have any leads here. Is it one particular part of the game that this is happening in?
